### ISSUE TITLE
Migratie to Trivy

### ISSUE THREAD

**Issue by yu-iskw** [reviewdog/action-tfsec#112]
## Overview
Thank you for the great Action. It enables us to keep good quality of terraform at scale. As you may know, tfsec was merged to Trivy. It would be great to use Trivy instead of tfsec. For instance,tfsec doesn't support new features in terraform 1.5+ any more. 

- https://github.com/aquasecurity/tfsec/discussions/1994
- https://github.com/aquasecurity/tfsec/issues/2070#issuecomment-1673920879

## Proposal
We may want to archive the repository and create a new repository copes with Trivy by reusing the great knowledge in the repository.

**Comment by shogo82148** [comment]
Your pull request is welcome!

**Comment by nayuta** [comment]
@shogo82148 Hi, I'm working on the issue and almost fixed other than README.md. Can I create a PR for this repository by fixing README.md?

**Comment by shogo82148** [comment]
@nayuta of course, you can. however I think https://github.com/nayuta/action-tfsec/pull/1 would be better to publish it as a new GitHub Action.

**Comment by shogo82148** [comment]
I created a new repository https://github.com/reviewdog/action-trivy based on https://github.com/nayuta/action-tfsec/pull/1

**Comment by nayuta** [comment]
@shogo82148 Thank you! However, there are some bugs in the trivy for Terraform. We may need to wait for the next update to use...

**Comment by shogo82148** [comment]
@nayuta Thanks! @review-dog invites you to the reviewdog organization; you can now create the `v1.0.0` tag on https://github.com/reviewdog/action-trivy

**Comment by nayuta** [comment]
@shogo82148 Thank you for inviting me. 

Could you review the other PR: https://github.com/reviewdog/action-trivy/pull/5? This PR will change behavior from the original action-tfsec, so it's ok to merge into the next upcoming release.

 I'll create a tag after hearing your answer.

**Comment by nayuta** [comment]
@shogo82148 I'm sorry for replying again. But I don't have permission to create a tag at `action-trivy`, but I have permission for this repo.

Could you fix this?

**Comment by shogo82148** [comment]
@nayuta I've updated the repository permission. could you try again?

**Comment by nayuta** [comment]
@shogo82148 Thank you! I've done!
